CHANGELOG.md

```diff
@@ -91,7 +96,13 @@ be found.
 - Fix an issue where Authz plugin would corrupt the payload body when faced with a large amount of data ([#20602](https://github.com/docker/docker/pull/20602))
 
 ### Runtime
-
+- Fix a panic that could occur when cleanup after a container started with invalid parameters [#21716](https://github.com/docker/docker/pull/21716)
+- Fix a race with event timers stopping early [#21692](https://github.com/docker/docker/pull/21692)
+- Fix race conditions in the layer store, potentially corrupting the map and crashing the process [#21677](https://github.com/docker/docker/pull/21677)
+- Un-deprecate auto-creation of host directories for mounts. This feature was marked deprecated in [#21666](https://github.com/docker/docker/pull/21666)
```

Comment:
you need to add two spaces at the end of this line to force a line-break in the rendered Markdown